.prettierrc.json

```
@@ -5,5 +5,15 @@
     "printWidth": 100,
     "singleQuote": false,
     "bracketSameLine": true,
-    "arrowParens": "avoid"
+    "arrowParens": "avoid",
+    "importOrder": [
+        "^react(.*)$",
+        "^next(.*)$",
+        "^@mayd/(.*)$",
```

And I guess we should escape `/`, shouldn't we?

```suggestion
        "^next(.*)$",
        "^@fr(e|a)ym\/(.*)$",
        "^@mayd\/(.*)$",
```